src/components/selectboxes.jsx

```diff
@@ -0,0 +1,52 @@
+
+var React = require('react');
+var valueMixin = require('./mixins/valueMixin.jsx');
+var multiMixin = require('./mixins/multiMixin.jsx');
+
+module.exports = React.createClass({
+    displayName: 'SelectBox',
+    mixins: [valueMixin, multiMixin],
```

Comment:
Select boxes do not need the multiMixin. That is for fields that can either be a single value or an array of values. Selectboxes are always an array of values so it is not needed.